fe1-web/src/resources/strings.ts

```diff
@@ -256,8 +256,17 @@ namespace STRINGS {
   export const lao_id = 'LAO ID';
   export const lao_name = 'LAO name';
   export const roll_call_name = 'Roll Call name';
+
+  /* --- Digital Cash Strings --- */
+  export const script_type = 'Pay-to-Pubkey-Hash';
+  export const coinbase_hash = 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=';
```

Comment:
Nitpick: this file looks to me to be human-readable interface text. It might be that translators even look at it. If that's the case, then those two constants would be more at home in `const.ts` because they are not meant to be understood by a human or translated.

Reply:
I agree, it might be a good time starting with this file. But to be fair, in the beginning of the semester and probably still at the moment `strings.ts` contains some constants mixed into the UI strings ... 😂

So while it is not optimal, it is consistent with the remaining code 😂

Reply:
I agree it's better to put it in `const.ts`. Even if it's not the case everywhere, we should clearly separate UI strings from constants.